tests/fixtures/saas/twilio_sms_fixtures.py

```diff
@@ -0,0 +1,117 @@
+from typing import Any, Dict, Generator
+
+import pydash
+import pytest
+import requests
+import time
+
+from requests.auth import HTTPBasicAuth
+from tests.ops.integration_tests.saas.connector_runner import (
+    ConnectorRunner,
+    generate_random_email,
+)
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("twilio_sms")
+
+
+@pytest.fixture(scope="session")
+def twilio_sms_secrets(saas_config) -> Dict[str, Any]:
+    return {
+        "domain": pydash.get(saas_config, "twilio_sms.domain") or secrets["domain"],
+        "twilio_account_sid": pydash.get(saas_config, "twilio_sms.twilio_account_sid") or secrets["twilio_account_sid"],
+        "twilio_auth_token": pydash.get(saas_config, "twilio_sms.twilio_auth_token")
+        or secrets["twilio_auth_token"],
+        "twilio_from_phone": pydash.get(saas_config, "twilio_sms.twilio_from_phone") or secrets["twilio_from_phone"],
+    }
+
+
+@pytest.fixture(scope="session")
+def twilio_sms_identity_phone_number(saas_config) -> str:
+    return (
+        pydash.get(saas_config, "twilio_sms.identity_phone_number") or secrets["identity_phone_number"]
+    )
+
+
+@pytest.fixture
+def twilio_sms_erasure_identity_email() -> str:
+    return generate_random_email()
+
+
+@pytest.fixture
+def twilio_sms_erasure_identity_phone_number() -> str:
+    # needs to be hardcoded as twilio restricts the use of phone numbers to valid and verified numbers - in this case this number is a virtual number generated by twilio NOTE this phone number could end up invalid depending on the status of our (connectors@ethyca.com) twilio account
+    return "+18777804236"
```

Comment:
Should this be injected via the connector's secrets instead of hardcoded? Sure it' not a _secret_, but it would feel cleaner.

Reply:
How's it look now?